Update Accessibility/TPP/XL-MS (#20)
dschust-r/review_ms_mem_prot: content/02.novel_mass_spec_methods.md

[@DOI:10.1021/acs.jproteome.3c00825; @DOI:10.1016/j.mcpro.2023.100588; @DOI:10.1038/s41592-020-01022-1; @DOI:10.1007/978-1-0716-3397-7_13; @DOI:10.1101/2025.04.28.650974; @DOI:10.1371/journal.pone.0282593; @DOI:10.1021/jacs.3c09078; @DOI:10.1038/s41596-021-00640-z; @DOI:10.1016/j.mcpro.2022.100206; @DOI:10.1021/acschembio.3c00781]

## 2. Accessibility approaches

> Accessibility approaches use chemical modifications or limited proteolysis to investigate solvent-exposed regions of proteins. 

Reviewer: ```suggestion
Surface accessibility is an increasingly popular metric that is used to study protein occupancy or conformations. Changes in surface accessibility can be indicative of conformational changes, protein-protein interactions,  or protein-ligand interactions. ```

> Changes in surface accessibility can be interpreted as conformational changes, protein interactions or effects of ligand binding. 

Reviewer: ```suggestion
Commonly used methods to assess the surface accessibility of one or multiple proteins include, among others, methods that introduce chemical modifications or pulse-proteolysis experiments.
```

> A key strength of these approaches is their applicability to low- and high-complexity samples. 

Reviewer: ```suggestion
These methods can be coupled to mass spectrometry, which makes them applicable to both low- and high-complexity samples. 
```

> However, in commonly used cell lysates, the samples are usually centrifuged prior to the experiment to remove unbroken cells and debris. 

Reviewer: ```suggestion

```

> This process also removes membrane fragments, resulting in a bias towards identifying soluble proteins. 

Reviewer: ```suggestion
```

> Adapted protocols have been developed to investigate changes in the accessibility of membrane proteins in either purified or native lipid environments.

Reviewer: ```suggestion
```

> Limited Proteolysis Coupled to Mass Spectrometry (LiP-MS) employs the broad-specificity protease proteinase K (PK) in a pulse-proteolysis approach to probe structural changes in proteins.

Reviewer: ```suggestion
Limited proteolysis-coupled mass spectrometry (LiP-MS) employs the broad-specificity protease proteinase K (PK) in a pulse-proteolysis approach to probe structural changes in proteins and complex protein mixtures.
```

> Because PK cleaves accessible, flexible regions, changes in proteolytic patterns can reveal conformational shifts in response to perturbations. This method has been applied to purified proteins, membrane-enriched fractions, and even crude lysates [@DOI:10.1007/978-1-0716-3397-7_13].

Reviewer: ```suggestion
In a typical LiP-MS experiment, a brief PK pulse-proteolysis step is followed by a tryptic digest and the generated peptide fragments are then analyzed with mass spectrometry. The generated structure-specific peptide fragments can then be compared across experimental conditions. This method has been applied to 
cellular lysates, purified soluble and membrane proteins, membrane-enriched fractions, and biofluids. [@DOI:10.1007/978-1-0716-3397-7_13].
```

> For example, in a study on Mycobacterium tuberculosis Cya, an enzyme involved in cAMP production, LiP-MS was used to characterize mutations within its intracellular binding pocket (Ec1). 

Reviewer: ```suggestion
In a study on the detergent-purified membrane protein Mycobacterium tuberculosis Cya, an enzyme involved in cAMP production, LiP-MS was used to characterize mutations within its intracellular binding pocket (Ec1). 
```

The results revealed altered protease accessibility and dynamics in the helical domain, providing structural insight into the effects of these mutations [@DOI:10.7554/eLife.77032]. 
In another application, LiP-MS was performed on detergent-free native bovine rod outer segment membranes to investigate regulation of the rod cyclic nucleotide-gated (CNG) channel by the calcium-binding protein calmodulin (CaM). 
The approach identified CaM-binding sites on the CNG channel and confirmed the formation of a ternary complex betweem the two CNG subunits and CaM [@DOI:10.1073/pnas.2300309120]. 

> Additionally, LiP-MS was used to analyze crude mammalian cell membranes, revealing that flexible domains are critical for modulating adenylate cyclase 8 (AC8) activity through interactions with CaM and G-proteins (Gα and Gβγ) [@DOI:10.1038/s44319-024-00076-y].

Reviewer: You could also (in one sentence) mention this study here: 10.1021/jacs.4c03922

This adaptation broadens the method’s applicability, including to membrane-associated proteins. 
For example, HT-PELSA successfully identified the primary target BCR-ABL1 as well as several known membrane-related off-targets upon dasatinib treatment. In another case, application to ampicillin-treated E. coli lysates revealed five established antibiotic targets, demonstrating the method’s utility also in microbial systems [@DOI:10.1101/2025.04.28.650974].
While protease-based methods such as LiP-MS and PELSA rely on differential cleavage to infer structural accessibility, other strategies use chemical labeling to probe solvent-exposed regions. 
Fast Photochemical Oxidation of Proteins (FPOP) uses hydroxyl radicals to label solvent-exposed amino acids but traditionally requires detergent purification, disrupting native environments. 

> While in-cell FPOP avoids detergents, water-soluble hydroxyl radicals provide poor transmembrane domain coverage. IC-FPOMP (in-cell fast photochemical oxidation of membrane proteins) addresses this using TiO₂ nanoparticles to generate reactive oxygen species near plasma membranes, improving transmembrane domain labeling in intact cells.

Reviewer: ```suggestion
While in-cell FPOP avoids detergents, water-soluble hydroxyl radicals provide poor transmembrane domain coverage. IC-FPOMP (in-cell fast photochemical oxidation of membrane proteins) addresses this by using TiO₂ nanoparticles to generate reactive oxygen species near plasma membranes, improving transmembrane domain labeling in intact cells.
```

For instance, IC-FPOMP was applied to the human glucose transporter GLUT1—an essential membrane protein in erythrocytes, the blood–brain barrier, and cancer cells. 
Comparisons between liposomal and native cellular membranes revealed conformation-dependent differences, highlighting how membrane composition can influence GLUT1 structure and dynamics. 
Moreover, IC-FPOMP enabled in-cell drug screening against GLUT1, revealing distinct protein painting on ligand binding [@DOI:10.1002/anie.202424779].

### Thermal proteome profiling

> Thermal Proteome Profiling (TPP) is a technique used to detect changes in protein thermal stability upon perturbations such as ligand binding. 

Reviewer: ```suggestion
Thermal Proteome Profiling (TPP) is a technique used to detect changes in protein thermal stability upon cellular perturbations, as well as ligand binding. 
```

Among these, the purinergic receptor P2X4 was identified as the primary target. This study provided the first direct evidence linking P2X4 to autophagy, showing that indophagolin acts as a P2X4 antagonist [@DOI: 10.1016/j.chembiol.2021.02.017].
To enhance resolution at the plasma membrane, a variant known as Cell Surface TPP (CS-TPP) was developed by integrating classical TPP with selective enrichment of biotinylated, surface-accessible proteins. 
For example, the action of the cardiac glycoside, ouabain, was examined using this method. 
While conventional TPP failed to detect its canonical target, CS-TPP successfully identified the Na⁺/K⁺-ATPase and revealed additional proteins potentially involved in its function or affected by off-target effects. 
The method has also been applied to investigate copper-induced cellular stress, identifying transporters associated with the high copper response, and to study T-cell receptor remodeling upon antibody engagement [@DOI: 10.1038/s41592-020-01022-1].

### Chemical cross-linking

> Crosslinking mass spectrometry (XL-MS) is a technique used to discover and characterize protein–protein interactions. 

Reviewer: ```suggestion
Cross-linking mass spectrometry (XL-MS) is a technique used to discover protein–protein interactions and identify binding-sites. 
```

> The principle involves the application of chemical compounds that covalently link specific functional groups on proteins via a spacer arm. 

Reviewer: ```suggestion
The method utilizes chemical compounds that covalently link specific functional groups on proteins via a spacer arm. The cross-linking reaction is typically followed by a tryptic digest, that is analyzed using mass spectrometry.
```

> The readouts from XL-MS experiments include intraprotein crosslinks (intralinks), which occur within a single protein, and interprotein crosslinks, which connect different proteins and provide spatial constraints.

Reviewer: ```suggestion
The readouts from XL-MS experiments include intraprotein cross-links (intralinks), which occur within a single protein, and interprotein cross-links, which connect different proteins. Both cross-link types provide spatial constraints and can be used to infer structural information.

Next to membrane and organelle enrichment strategies, newer approaches have expanded XL-MS using unconventional crosslinking chemistries. 
One such method targets glycan-mediated interactions by labeling surface glycans with azido groups and crosslinking them to nearby primary amines. 
This enables identification of glycan-dependent crosslinks, providing insights into glycosylation sites, glycoforms, and interaction patterns on both the extracellular and cytosolic faces of membranes. 
Using this strategy, interaction networks involving glycosylated integrins, aminopeptidase N, and lysosome-associated membrane proteins have been described [@DOI: 10.1039/d1sc00814e].
Overall, XL-MS complements classical structural methods by capturing protein–protein interactions in flexible or dynamic regions. 
In complex biological systems such as the plasma membrane or organelles, it enables the mapping of interaction hubs and characterization of membrane-associated proteins. 
Emerging chemistries, including glycan-targeted crosslinking, further broaden its utility by revealing modification-dependent interactions.

> ##Crosslink-Guided Modeling of Protein Structures

Reviewer: I think this could be nice to add. Would leave it here for now, with the option to add it to the outlook later on.

Beyond identifying interaction partners and mapping topologies, XL-MS data can also be integrated into structural modeling workflows to refine or predict protein complex architectures. 
Crosslink-derived distance restraints are increasingly used to guide docking and structural modeling, especially for membrane proteins where experimental structures remain limited. Tools such as HADDOCK (High Ambiguity Driven Biomolecular DOCKing) enable the incorporation of crosslinking data to model protein–protein interactions with higher spatial accuracy.

> For example, docking guided by XL-MS and further analyzed with DisVis revealed three distinct interfaces between COX and AIFM1, enabling the modeling of a COX–AIFM1 complex. 

Reviewer: AC8-Gbg was also modeled using XL-MS data, DisVis and HADDOCK

This provided insight into how AIFM1 may insert into the inner mitochondrial membrane and suggested potential functional implications [@DOI: 10.1073/pnas.2106950118]. 
More recently, crosslinking data have also been integrated with AlphaFold3 predictions for membrane protein predictions (AF3x). In the case of the SLC19A3 transporter, AlphaFold3 alone incorrectly predicted the complex with a nanobody in its inward-open state, whereas the inclusion of crosslinking restraints led to a model more consistent with the experimentally resolved cryo-EM structure [@DOI: https://doi.org/10.1101/2024.12.03.626671].
Altogether, crosslink-guided modeling provides a bridge between experimental data and structural prediction. 
Especially for membrane proteins—where high-resolution structures remain scarce—it offers a way to build and test mechanistic hypotheses grounded in experimental proximity data.

> 

Reviewer: The section is nicely written, a good summary of recent advances in the field. Do you think we could maybe make a better connection to pharmacology? Maybe add some sentences in different parts or highlighting drug interaction screens? Thoughts?
